Weird CI errors:

```
In file included from /workdir/zephyr/subsys/logging/log_core.c:6:
../../../../../../include/logging/log_msg.h: In function ‘z_log_msg_std_alloc’:
../../../../../../include/logging/log_msg.h:330:7: warning: implicit declaration of function ‘IS_ENABLED’ [-Wimplicit-function-declaration]
  330 |   if (IS_ENABLED(CONFIG_USERSPACE)) {
      |       ^~~~~~~~~~
../../../../../../include/logging/log_msg.h:330:18: error: ‘CONFIG_USERSPACE’ undeclared (first use in this function); did you mean ‘CONFIG_BT_RPA’?
  330 |   if (IS_ENABLED(CONFIG_USERSPACE)) {
      |                  ^~~~~~~~~~~~~~~~
      |                  CONFIG_BT_RPA
```

--- a/include/logging/log_msg.h
+++ b/include/logging/log_msg.h
@@ -7,6 +7,7 @@
 #define ZEPHYR_INCLUDE_LOGGING_LOG_MSG_H_
 
 #include <sys/atomic.h>
+#include <sys/util.h>
 #include <string.h>
 
 #ifdef __cplusplus
@@ -326,6 +327,13 @@ static inline struct log_msg *z_log_msg_std_alloc(void)
 		msg->hdr.ref_cnt = 1;
 		msg->hdr.params.raw = 0U;
 		msg->hdr.params.std.type = LOG_MSG_TYPE_STD;
+
+		if (IS_ENABLED(CONFIG_USERSPACE)) {
+			/* it may be used in msg_free() function. */
+			msg->hdr.ids.level = 0;
+			msg->hdr.ids.domain_id = 0;
+			msg->hdr.ids.source_id = 0;
+		}
 	}
 
 	return msg;

